scripts/kill_horse.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+test $(uname -m) != "x86_64" && exit 0
+
+LICENSE_FILE=/home/pi/.firewalla/license
+
+touch $LICENSE_FILE
+
+#test -e $LICENSE_FILE && exit 0
+#
+#check_ethtool() {
+#    pgrep -x ethtool &>/dev/null && return 1
+#    sleep 0.3
+#    pgrep -x ethtool &>/dev/null && return 1
+#    sleep 0.3
+#    pgrep -x ethtool &>/dev/null && return 1
+#    return 0
+#}
+#
+## exit if not exists
+#check_ethtool && exit 0
+#
+## double check
+#test -e $LICENSE_FILE && exit 0
+#
+## race condition probability is low
+#touch $LICENSE_FILE
+#sudo pkill -x ethtool
+#
+## harmless, no need to rm
+##sleep 1
+##test -s $LICENSE_FILE || rm $LICENSE_FILE
```

Comment:
Why not just remove these lines rather than comment them out?

Reply:
becoz i may need to take it back in the future